src/pyspiffe/bundle/x509_bundle/x509_bundle.py

```diff
@@ -158,8 +158,8 @@ def load(
         if encoding == serialization.Encoding.DER:
             return cls.parse_raw(trust_domain, bundle_bytes)
 
-        raise ValueError(
-            'Encoding not supported: {}. Expected \'PEM\' or \'DER\'.'.format(encoding)
+        raise ValidationError(
```

Comment:
This error is thrown when a not supported (aka illegal) encoding is passed, so it would raise a `IllegalArgumentError`